### DISCUSSION TITLE
Several independent SE inverters

### DISCUSSION THREAD

**Discussion by FranciscoBigWings** [WillCodeForCats/solaredge-modbus-multi#304] [General]
**Describe the feature**
Several independent inverters without leader/follower configuration with individual internal IPs.

**Additional context**
I have a more than complex solar configuration besides having two SE inverters. I control my export/import with an external meter (SE doesnt read from it) so basically my SE meter is useless for data logging but it was still usefull to control the site limit power. My problem now is that I need to "break" the leader/follower connection between inverters and make them as they were both individual.

How could I keep using the integration with two different inverters? The integration always tries to find the second inverter as it was a follower inverter but I cant search for a second internal IP instead.

I hope I explained myself well

Thanks

**Comment by WillCodeForCats** [comment]
Add the integration multiple times for each IP address.

Edit: And you'll have to either delete the existing integration that was set up with 2 inverters (or edit `/config/.storage/core.config_entries`) because HA doesn't allow re-entering the integration setup step to change the number of inverters value from 2 to 1.
